package/doc/sphinx/source/documentation_pages/analysis/parallelization.rst

```diff
@@ -0,0 +1,360 @@
+parallelization with the :class:`MDAnalysis.analysis.base.AnalysisBase`, what 
+users and developers need to know, when you should use parallelization (almost 
+always!), and when you should abstain from doing so (rarely).
+
+
+How to use parallelization
+==========================
+
+In order to use parallelization in a built-in analysis class ``SomeClass``,
+simply check which backends are available (see :ref:`backends` for backends
+that are generally available), and then just enable them by providing
+``backend='multiprocessing'`` and ``n_workers=...`` to ``SomeClass.run(...)``
+method:
+
+.. code-block:: python
+
+    u = mda.Universe(...)
+    my_run = SomeClass(trajectory)
+    assert SomeClass.get_supported_backends() == ('serial', 'multiprocessing', 'dask')
+
+    my_run.run(backend='multiprocessing', n_workers=12)
+
+For some classes, such as :class:`MDAnalysis.analysis.rms.RMSF` (in its current implementation),
+split-apply-combine parallelization isn't possible, and running them will be
+impossible with any but the ``serial`` backend.
+
+.. Note::
+
+   Parallelization is getting added to existing analysis classes.  Initially,
+   only :class:`MDAnalysis.analysis.rms.RMSD` supports parallel analysis, but
+   the goal is that the 3.0 release will support parallel analysis where
+   possible.
+
+
+How does parallelization work
+=============================
+
+The main idea behind its current implementation is that a trajectory analysis is
+often trivially parallelizable, meaning you can analyze all frames
+independently, and then merge them in a single object. This approach is also
+known as "split-apply-combine", and isn't new to MDAnalysis users, since it was
+first introduced in `PMDA`_ :footcite:p:`Fan2019`.  
+Version 2.8.0 of MDAnalysis brings this approach to the main library.
+
+.. _`PMDA`: https://github.com/mdanalysis/pmda
+
+
+split-apply-combine
+-------------------
+
+The following scheme explains the current :meth:`AnalysisBase.run
+<MDAnalysis.analysis.base.AnalysisBase.run>` protocol (user-implemented methods
+are highlighted in orange):
+
+.. figure:: /images/AnalysisBase_parallel.png
+
+
+In short, after checking input parameters and configuring the backend,
+:class:`AnalysisBase <<MDAnalysis.analysis.base.AnalysisBase>` splits all the
+frames into *computation groups* (equally sized sequential groups of frames to
+be processed by each worker). All groups then get **split** between workers of
+a backend configured early, the main instance gets serialized and distributed
+between workers, and then
+:meth:`~MDAnalysis.analysis.base.AnalysisBase._compute()` method gets called
+for all frames of a computation group. Within this method, a user-implemented
+:meth:`~MDAnalysis.analysis.base.AnalysisBase._single_frame` method gets
+**applied** to each frame in a computation group.  After that, the main
+instance gets an object that will **combine** all the objects from remote
```

Comment:
```suggestion
instance gets an object that will **combine** all the objects from other
```
I think "remote" only applied to `dask-distributed` that was finally dropped for this PR?